cerberus/cerberus.py

```diff
@@ -758,6 +758,17 @@ def _validate_valueschema(self, schema, field, value):
                 if len(validator.errors):
                     self._error(field, validator.errors)
 
+    def _validate_excludes(self, exclude, field, value):
+        targets = []
```

Comment:
it's enough to wrap a single string into a list with the name `excludes`, no need for an extra object `target` imo.